setup.py

```diff
@@ -34,5 +34,9 @@ def get_version():
                      "can be copied or flashed a lot faster with bmaptool "
                      "than with traditional tools like \"dd\" or \"cp\". See "
                      "source.tizen.org/documentation/reference/bmaptool for "
-                     "more information."
+                     "more information.",
+    classifiers=[
+        "Programming Language :: Python :: 2.7",
+        "Programming Language :: Python :: 3"
```

Comment:
Does it make sense to have 3.3+ ? From brief look at python.org release pages, only 2.7 and 3.4+ are maintained currently.

Reply:
https://www.python.org/dev/peps/pep-0398/#x-end-of-life